lib/glific/flows/flow.ex

```diff
@@ -411,6 +412,50 @@ defmodule Glific.Flows.Flow do
       else: [dangling: "Your flow has dangling nodes"] ++ errors
   end
 
+  @spec check_flow_keywords(Keyword.t(), map()) :: Keyword.t()
+  defp check_flow_keywords(errors, flow) do
+    all_nodes = flow_objects(flow, :node)
+
+    wait_for_response_words =
+      all_nodes
```

Comment:
This is a very expensive function we should clean it.

Reply:
I have removed this and handling it in `Case.validate`